src/controllers/update/update.confirmation.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+import * as config from "../../config";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${config.UPDATE_CONFIRMATION_URL}`);
+    return res.render(config.UPDATE_CONFIRMATION_PAGE, {
+      backLinkUrl: config.UPDATE_LANDING_PAGE_URL,
```

Comment:
UPDATE_LANDING_PAGE_URL is incorrect for the back link as that's the first page in the journey, but not sure what your back link should be. Probably something to check with David or Matthew from product.